src/actions/GraphDataActions.ts

```diff
@@ -12,7 +13,7 @@ export const GraphDataActions = {
   getGraphDataStart: createAction(GraphDataActionKeys.GET_GRAPH_DATA_START),
   getGraphDataSuccess: createAction(
     GraphDataActionKeys.GET_GRAPH_DATA_SUCCESS,
-    resolve => (timestamp: number, graphDuration: number, graphData: any) =>
+    resolve => (timestamp: number, graphDuration: number, graphData: GraphElements) =>
```

Comment:
Can we also change `timestamp: TimeInMilliseconds` and `duration: DurationInSeconds`

Reply:
Changed!